test/widgets/message_list_test.dart

```diff
@@ -135,19 +135,6 @@ void main() {
       final padding = MediaQuery.of(element).padding;
       check(padding).equals(EdgeInsets.zero);
     });
-
-    testWidgets('content in MentionsNarrow not asked to consume insets (including bottom)', (tester) async {
```

Comment:
Oh one other commit-message comment:

    A test on CombinedFeedNarrow is sufficient for preventing this particular
    regression. We could go further and test this on other narrows, but
    there is little to gain other than verifying that those other narrows do
    not have a compose box, which is covered statically with the exhaustive
    check from hasComposeBox.

The last sentence here doesn't sound right, as this test didn't check that the narrow doesn't have a compose box. Rather, if the narrow had a compose box, the test would pass even if the bug were to return on other narrows that don't have the compose box. The original bug reproduced only on CombinedFeedNarrow (the only narrow lacking a compose box at the time), which is why the repro test used that narrow.